### DISCUSSION TITLE
Remove color indicator

### DISCUSSION THREAD

**Discussion by ronkristoff** [zed-industries/zed#14632] [Feature Requests]
How do i remove the color group indicator on the editor?
![Screenshot 2024-07-17 at 4 29 09 PM](https://github.com/user-attachments/assets/71be5eda-e262-4aed-87e7-36a0391bcfda)

**Comment by tepavcevic** [comment]
You can turn it off in the `settings.json`. Just add:

```json
"git": {
    "git_gutter": "hide"
  }
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
